signer/tests/integration/stacks_events_observer.rs

```diff
@@ -109,7 +109,7 @@ async fn test_new_blocks_sends_update_deposits_to_emily() {
         .expect("Wiping Emily database in test setup failed.");
 
     let body = COMPLETED_DEPOSIT_WEBHOOK.to_string();
-    let new_block_event = serde_json::from_str::<NewBlockEvent>(&body).unwrap();
+    serde_json::from_str::<NewBlockEvent>(&body).unwrap();
```

Comment:
nit: remove this?

Reply:
I kept it because I want to check that we deserialize the JSON correctly. I suppose that test is implicitly handled in this test anyway, and it is handled in our unit tests.

Reply:
I'll just remove it

Reply:
Removed in https://github.com/stacks-network/sbtc/pull/1115/commits/5a352d6912b9512f87c044c1dbb22f41c69cc950